packages/kit/src/components/TabPageHeader/HeaderRight.tsx

```
@@ -171,35 +140,34 @@ export function HeaderRight({
     badge,
+    children,
     devSettings.enabled,
     devSettings?.settings?.showPrimeTest,
     firstTimeGuideOpened,
     intl,
+    media.gtMd,
     isShowGtMdItem,
     onScanButtonPressed,
     openNotificationsModal,
     sceneName,
   ]);
```

ESLint

[error] React Hook useMemo has unnecessary dependencies: 'isShowGtMdItem' and 'onScanButtonPressed'. Either exclude them or remove the dependency array. Outer scope values like 'onScanButtonPressed' aren't valid dependencies because mutating them doesn't re-render the component.

(react-hooks/exhaustive-deps)